### PR TITLE
Product list edit

### PR DESCRIPTION
**Pull request by thecrewmate876**
CC @wgyt @RedGuy12

### CODE REVIEW

File: pages/index.vue [hunk 1]
```diff
@@ -64,11 +64,11 @@
         <img
           src="/brand/OneApi.svg"
           class="object-fill h-16 rounded-lg rounded-b-none"
-          alt="OneApi"
+          alt="OneAPI"
         />
         <div class="prose">
           <h2>
-            OneApi will be a package for 1host that makes making apis easy
+            OneAPI will be a package for 1Host that will make erecting APIs easy
```
Comment (RedGuy12) [inline pages/index.vue:71]: 1host is lowercase

File: pages/index.vue [hunk 2]
```diff
@@ -149,8 +149,8 @@
           alt="OneCDN"
         />
         <div class="prose">
-          <h2>OneCDN is a CDN that serves static files</h2>
-          <a href="https://github.com/onedotprojects/cdn">Github</a>
+          <h2>OneCDN is a CDN used by OneDot products that serves static files</h2>
```
Comment (RedGuy12) [inline pages/index.vue:152]: take out ` used by OneDot products`

File: pages/index.vue [hunk 3]
```diff
@@ -223,12 +223,12 @@
         <img
           src="/brand/1host.svg"
           class="object-fill h-16 rounded-lg rounded-b-none"
-          alt="1host"
+          alt="1Host"
```
Comment (RedGuy12) [inline pages/index.vue:226]: lowercase

File: pages/index.vue [hunk 4]
```diff
@@ -223,12 +223,12 @@
...
         />
         <div class="prose">
-          <h2>An easy way to host websites using modules</h2>
-          <a href="https://1host.js.org/">Link</a>
-          <a href="https://github.com/1hostjs/hosting">Github</a>
+          <h2>1Host is an easy way to host websites using modules</h2>
```
Comment (RedGuy12) [inline pages/index.vue:229]: lowercase

File: pages/index.vue [hunk 5]
```diff
@@ -223,12 +223,12 @@
...
+          <a href="https://1host.js.org/">Documentation</a>
+          <a href="https://github.com/1hostjs/hosting">GitHub repo</a>
```
Comment (RedGuy12) [inline pages/index.vue:231]: add link to npm?

File: pages/index.vue [hunk 6]
```diff
@@ -253,7 +253,7 @@
           alt="OneMarkdown"
         />
         <div class="prose">
-          <h2>Something like markdown but supports all HTML tags</h2>
+          <h2>OneMarkdown will be a custom flavor of Markdown that supports all HTML tags</h2>
```
Comment (RedGuy12) [inline pages/index.vue:256]:
```suggestion
          <h2>OneMarkdown will be a Markdown flavor that supports more than other flavors</h2>
```

File: pages/index.vue (hunk 1 above)
Comment (thecrewmate876) [reply pages/index.vue:71]: > 
> 
> 1host is lowercase

@RedGuy12 we use both so it doesn't matter

File: pages/index.vue (hunk 2 above)
Comment (thecrewmate876) [reply pages/index.vue:152]: > 
> 
> take out ` used by OneDot products`

@RedGuy12 why it's only used by onedot rn

File: pages/index.vue (hunk 3 above)
Comment (thecrewmate876) [reply pages/index.vue:226]: > 
> 
> lowercase

@RedGuy12 https://github.com/onedotprojects/1dot-website/pull/7#discussion_r693196829

File: pages/index.vue (hunk 4 above)
Comment (thecrewmate876) [reply pages/index.vue:229]: > 
> 
> lowercase

@RedGuy12 https://github.com/onedotprojects/1dot-website/pull/7#discussion_r693196829

File: pages/index.vue (hunk 5 above)
Comment (thecrewmate876) [reply pages/index.vue:231]: > 
> 
> add link to npm?

@RedGuy12 ok i'll do that

File: pages/index.vue (hunk 6 above)
Comment (thecrewmate876) [reply pages/index.vue:256]: @wgyt what do you think

File: pages/index.vue (hunk 1 above)
Comment (RedGuy12) [reply pages/index.vue:71]: no?
image

File: pages/index.vue (hunk 2 above)
Comment (RedGuy12) [reply pages/index.vue:152]: nobody uses it
we use cf pages now

**Comment by retronbv** [comment]
Idea: fetch list of repos from GitHub api and load it instead of hard coding it

**Pull request merged**